src/utilities/customHooks.jsx

```diff
@@ -0,0 +1,10 @@
+import { useState } from 'react';
+
+// eslint-disable-next-line import/prefer-default-export
+export const useInput = function useFormInputWithUseState(initialValue) {
+  const [value, setValue] = useState(initialValue);
+  return [
+    { value, onChange: (e) => setValue(e.target.value) },
+    () => setValue(initialValue),
```

Comment:
The first element of this array is a bit confusing. Can you clarify what it's returning? It seems like an object with a value and a onChange property maybe?

Reply:
Haha, thank you! The textbook explains it very clearly!